lms/djangoapps/certificates/tests/test_queue.py

```diff
@@ -73,6 +78,58 @@ def test_no_create_action_in_queue_for_html_view_certs(self):
         self.assertEqual(certificate.status, CertificateStatuses.downloadable)
         self.assertIsNotNone(certificate.verify_uuid)
 
+    @ddt.data(
+        ('honor', 'honor'),
```

Comment:
Your data can be simplified to a single list of strings, since they are both the same.